### DISCUSSION TITLE
Use Virtua with Lit Element

### DISCUSSION THREAD

**Discussion by clibu** [inokawa/virtua#418] [Ideas]
I'd like to use virtua with [Lit Element](https://lit.dev/) I've read through the docs for use with React, Vue & Solid but can't see how I can adapt any of these examples to work with Lit. Lit is based around Web Components, so having virtua support Lit & therefore Web components would be great. 

Thanks for all your work on virtua.

**Comment by inokawa** [comment]
I've created a new issue for it #419 .
Please +1 it if anyone visited here is interested in it.
